src/components/carousel/panels/dataset-info-panel.js

```diff
@@ -0,0 +1,33 @@
+import React, { Fragment } from 'react'
+import { DetailsPanel, DetailWrapper, Headline, Description } from './subcomponents/LeftPanel'
+import { DataPanel, DataWrapper, CarouselListItem, CarouselBulletedList } from './subcomponents/RightPanel'
+import { datasetInfoContent } from '../types'
+import { useWindowWidth } from "../../../hooks";
+
+export const DatasetInfoPanel = ({ content }) => {
+  const { isCompact } = useWindowWidth();
+
+  return (
+    <Fragment>
+      <DetailsPanel compact={isCompact}>
+        <DetailWrapper dataset compact={isCompact}>
+          <Headline compact={isCompact}>{content.headline}</Headline>
+          <Description compact={isCompact}>{content.description}</Description>
+        </DetailWrapper>
+      </DetailsPanel>
+      <DataPanel dataset compact={isCompact}>
+        <DataWrapper compact={isCompact}>
+          <CarouselBulletedList compact={isCompact}>
+            {content.datasets.map((datum, i) => (
+              <CarouselListItem key={i} compact={isCompact}>{datum.name}</CarouselListItem>
```

Comment:
i see potential for duplicate keys ending up in the same view with this. consider a more specialized, carousel-specific key naming